pkg/scheduler/apis/config/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- dixudx
```

Comment:
kubernetes/api-approvers should be here, as well as sig-scheduling leads. I'm not sure I would add you quite yet as that wasn't mentioned in the KEP